src/ops/math.jl

```diff
@@ -409,3 +409,15 @@ for reduction in [:sum, :prod, :min, :max, :mean]
         Tensor(Operation(desc), 1)
     end
 end
+
```

Comment:
Are there other segment operations? I could not find any except segment_sum and others defined just above my code.